.github/workflows/test.yml

```diff
@@ -50,3 +50,49 @@ jobs:
       if: github.event_name == 'push' && github.ref == 'refs/heads/master'
       env:
         FUZZIT_API_KEY: ${{ secrets.FUZZIT_API_KEY }}
+
+  docker:
```

Comment:
I prefer this to be part of a `deploy.yaml` workflow, as it's not really related to test, and only runs on pushes.

Reply:
One hitch with this approach in Github Actions though: AFAIK jobs in one foo.yml can'd depend on jobs in another bar.yml, so the deployment couldn't be conditional on the tests passing.

Reply:
Ah, that's a good point. SGTM

Reply:
I took you SGTM as meaning you're OK with having it in test.yml, conditioned on the tests passing. If I misunderstood let me know.